src/use_case_classification.Rmd

```diff
@@ -0,0 +1,147 @@
+# Use Cases
+
+## Classification
+
+One popular example regarding a classification task is the "Titanic" showcase. We have different passenger information - like name, age or fare - available with the aim to predict which kind of people would have survived the titanic sinking.
+
+Therefore we load the titanic dataset and other libraries that are needed for this use case.
+
+```{r, results='hide', message=FALSE, warning=FALSE}
+library(titanic)
+library(mlr)
+library(BBmisc)
+```
+
+
+```{r}
+data = titanic_train
+head(data)
+```
+
+Our aim - as mentioned before - is to predict which kind of people would have survided.
+
+Therefore we will work off the following steps:
+
+* preprocessing, [here](http://mlr-org.github.io/mlr-tutorial/devel/html/preproc/index.html) and [here](http://mlr-org.github.io/mlr-tutorial/devel/html/impute/index.html)
+* create a task,[here](http://mlr-org.github.io/mlr-tutorial/devel/html/task/index.html)
+* provide a learner, [here](http://mlr-org.github.io/mlr-tutorial/devel/html/learner/index.html)
+* train the model, [here](http://mlr-org.github.io/mlr-tutorial/devel/html/train/index.html)
+* predict the survival chance, [here](http://mlr-org.github.io/mlr-tutorial/devel/html/predict/index.html)
+* validate the model,[here](http://mlr-org.github.io/mlr-tutorial/devel/html/performance/index.html) and [here](http://mlr-org.github.io/mlr-tutorial/devel/html/resample/index.html)
+
```

Comment:
In general I like this list.
But in my opinion we could make it a little less technical. Things like "create a task" or "provide a learner" won't necessarily mean sth. to someone new to mlr. Therefore, I would change these lines to sth. like "Define the learning task/problem" and "select a learning method"